### PR TITLE
Preserve linebreaks in only-comment blocks

### LINKED ISSUE #506: Preserve linebreaks in only-comment code blocks
Input:
```php
<?php
//A

// B
?>
<h1>Test</h1>
```
Output:
```php
<?php
//A
// B
?>
<h1>Test</h1>
```

### PR DESCRIPTION
**Pull request by czosel**
Fixes #506

### CODE REVIEW

File: tests/comments/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -1304,6 +1306,7 @@ exports[`inline6.php 1`] = `
 /**
  * test
  */
+
```
Comment (czosel) [inline tests/comments/__snapshots__/jsfmt.spec.js.snap]: This should probably go, right?

File: src/comments.js [hunk 2]
```diff
@@ -278,14 +278,7 @@ const handleAlternate = comment => {
...
   const { precedingNode, enclosingNode, followingNode } = comment;
-  if (
-    !enclosingNode &&
-    followingNode &&
-    !precedingNode &&
-    followingNode.kind === "inline"
-  ) {
-    // addLeadingComment(followingNode, comment);
-    comment.printed = true;
+  if (!enclosingNode && followingNode && followingNode.kind === "inline") {
```
Comment (czosel) [inline src/comments.js]: This is just a clean-up that i meant to already in the previous PR.

File: tests/comments/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (evilebottnawi) [reply tests/comments/__snapshots__/jsfmt.spec.js.snap]: I think no, but it is not related to this PR, we insert hardline before last closing tag (if newline exists in source code), and here we have two newlines, better create new issue

Comment (czosel) [reply tests/comments/__snapshots__/jsfmt.spec.js.snap]: @evilebottnawi fixed it!

File: src/printer.js [hunk 3]
```diff
@@ -1335,6 +1335,21 @@ function printLines(path, options, print, childrenAttribute = "children") {
     }
 
     if (isInlineNode) {
+      const printComments = comments => {
```
Comment (evilebottnawi) [inline src/printer.js]: Let's move this function to `comments.js`

Comment (czosel) [reply src/printer.js]: Good idea :+1: Also refactored a little.

**Pull request merged**